utils/library_conversion_helper.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2022 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef GOOGLE_SERVICE_CONTROL_CLIENT_UTILS_LIBRARY_CONVERSION_HELPER_H_
+#define GOOGLE_SERVICE_CONTROL_CLIENT_UTILS_LIBRARY_CONVERSION_HELPER_H_
+
+#include "google/rpc/status.pb.h"
+
+namespace google {
+namespace service_control_client {
+namespace library_conversion_helper {
```

Comment:
name space should be consistent with other files.   just google::service_control_client

Reply:
Done.